### DISCUSSION TITLE
tootctl feeds build

### DISCUSSION THREAD

**Discussion by adrianomic** [mastodon/mastodon#17319] [General]
What code does mastodon execute when run command ``tootctl feeds build``?

**Comment by Gargron** [comment]
https://github.com/mastodon/mastodon/blob/main/lib/mastodon/feeds_cli.rb#L26

**Comment by adrianomic** [comment]
@Gargron I want to ask some questions.
If someone delete their post, then our code delete that post actually from db instead of flagging them as deleted.
Am I right? If yes, then can you please explain me some about it?

Because I found some weird follows. If someone delete post, then post's follows should be deleted. By they're live.
And they're raising issue when run ``tootctl feeds build``

Let me know what you think about this

Thank you
